tests/benchmark/benchmark_helpers.go

```diff
@@ -0,0 +1,661 @@
+// Copyright 2022 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package benchmark
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"testing"
+
+	"github.com/apigee/registry/connection"
+	"github.com/apigee/registry/rpc"
+	"google.golang.org/genproto/protobuf/field_mask"
+)
+
+var (
+	registryProject string
+	rootResource    string
+)
+
+func init() {
+	flag.StringVar(&registryProject, "registry_project", "bench", "Name of the Registry project")
+	rootResource = fmt.Sprintf("projects/%s/locations/global", registryProject)
+}
+func getApiName(i int) string {
+	return fmt.Sprintf("test-%d", i)
+}
+
+func createGrpcClient(b *testing.B) (error, rpc.RegistryClient, context.Context) {
+	b.Helper()
+	ctx := context.Background()
+	gapicClient, err := connection.NewClient(ctx)
+	if err != nil {
+		b.Fatalf("Setup: Failed to create client: %s", err)
+		return err, nil, nil
+	}
+	client := gapicClient.GrpcClient()
+	b.StopTimer()
+	b.ResetTimer()
+	return nil, client, ctx
+}
+
+func CreateApi(apiId string, b *testing.B, record bool, client rpc.RegistryClient, ctx context.Context) *rpc.Api {
```

Comment:
I didn't find any calls of this function where `record` is set to `true`. I tend to think that this is more granularity than necessary -- I would instead just make sure that measurement is off during initialization and teardown, which could be done with calls that start/stop the timer in the main test function. Why might we keep this?